.gitignore

```diff
@@ -2,5 +2,6 @@
 .idea/vcs.xml
 
 out*
-
-c.out
\ No newline at end of file
+*.out
+istio-*/
```

Comment:
Remove these?

Reply:
Sure. Had added these here as they are artifacts from the setup scripts though can remove them as they aren't needed for this PR.